submissions/DaniaB24/a-tiny-js-world/main.js

```diff
@@ -0,0 +1,41 @@
+const cat = {
+  species: 'cat',
+  name: 'Mars',
+  hands: undefined,
+  gender: 'male',
+  legs: 4,
+  saying: 'mmeeow',
+}
+const dog = {
+  species: 'dog',
+  name: 'Lord',
+  hands: undefined,
+  gender: 'male',
+  legs: 4,
+  saying: 'mmeeow',
+}
+const man = {
+  species: 'human',
+  name: 'Ron',
+  hands: 2,
+  gender: 'male',
+  legs: 2,
+  saying: 'ahoj!',
+}
+const women = {
+  species: 'human',
+  name: 'Hermione',
+  hands: 2,
+  gender: 'female',
+  legs: 2,
+  saying: 'hello!',
+}
+
+const creatures = [cat, dog, man, women];
+const key = ['species', 'name', 'hands', 'legs', 'gender', 'saying']
+
+// ======== OUTPUT ========
+print(cat.species + ';' + cat.name + ';' + cat.hands + ';' + cat.gender + ';' + cat.legs + ';' + cat.saying);
+print(dog.species + ';' + dog.name + ';' + dog.hands + ';' + dog.gender + ';' + dog.legs + ';' + dog.saying);
+print(man.species + ';' + man.name + ';' + man.hands + ';' + man.gender + ';' + man.legs + ';' + man.saying);
+print(women.species + ';' + women.name + ';' + women.hands + ';' + women.gender + ';' + women.legs + ';' + women.saying);
```

Comment:
(1) Double-check the tem one of the checklist above (re DRY code).
Ask questions if anything is not clear - here or on Students' chat.

(2) Please tune your IDE to add an empty line at the end of files.
![image](https://user-images.githubusercontent.com/8686631/185761662-8c4345fd-e833-4d86-a2fd-b592efb983ce.png)

Reply:
its also pre-oop code , sorry:( I will change the link to the code

Reply:
Ping me when you double-check everything.
I'd expect to read something like "Listed requirements are met, questions I had are asked, code is updated accordingly, links are updated".

Reply:
I think links are updated @OleksiyRudenko 

> Ping me when you double-check everything. I'd expect to read something like "Listed requirements are met, questions I had are asked, code is updated accordingly, links are updated".

Reply:
I still can see that there is no new line at the end of file:
![image](https://user-images.githubusercontent.com/8686631/185972934-13810799-4b5a-45d0-9fe5-080cffe76f90.png)

Also there is a requirement which says:
- [ ] Code is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), which means that whenever you see a pattern in your code those should be eliminated as much as possible. Examples:
    - `print(dog); print(cat); etc ... ` should be refactored employing `Array.forEach` as the least
    - `` `${obj.legs}; ${obj.name}; etc...` `` (yes, strings are also code) must be refactored employing appropriate `Array` methods

Feel free asking questions.

Reply:
> I still can see that there is no new line at the end of file: ![image](https://user-images.githubusercontent.com/8686631/185972934-13810799-4b5a-45d0-9fe5-080cffe76f90.png)
> 
> Also there is a requirement which says:
> 
> * [ ]  Code is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), which means that whenever you see a pattern in your code those should be eliminated as much as possible. Examples:
>   
>   * `print(dog); print(cat); etc ... ` should be refactored employing `Array.forEach` as the least
>   * `` `${obj.legs}; ${obj.name}; etc...` `` (yes, strings are also code) must be refactored employing appropriate `Array` methods
> 
> Feel free asking questions.

I honestly do not know why this happens, I will try to fix it tomorrow

Reply:
> I still can see that there is no new line at the end of file: ![image](https://user-images.githubusercontent.com/8686631/185972934-13810799-4b5a-45d0-9fe5-080cffe76f90.png)
> 
> Also there is a requirement which says:
> 
> * [ ]  Code is [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), which means that whenever you see a pattern in your code those should be eliminated as much as possible. Examples:
>   
>   * `print(dog); print(cat); etc ... ` should be refactored employing `Array.forEach` as the least
>   * `` `${obj.legs}; ${obj.name}; etc...` `` (yes, strings are also code) must be refactored employing appropriate `Array` methods
> 
> Feel free asking questions.

I think I did it))) review pls @OleksiyRudenko